ruby_on_rails/configure_ci.md

```diff
@@ -1,8 +1,42 @@
 # Configure the CI
 
-It's now time to configure our CI to run the tests.
-The command to run the tests is `bin/check`.
-You can follow the section about [Configuring a CI](../configure_ci.md) to setup it properly on [SemaphoreCI](https://semaphoreci.com).
+At Renuo we **always** use a CI (Continuous Integration) system to test our applications. It's essential to guarantee
+that all the tests pass before building and releasing a new version through our CD system. Our projects use
+[SemaphoreCI](<https://semaphoreci.com/>).
+
+Before configuring the CI, you should already have a Git Repository with the code, a `bin/check` command to execute,
+and the main branches already pushed and ready to be tested.
+
+* Proceed to <https://semaphoreci.com/> and login or create an account with your Renuo email address.
+You should already be part of the Renuo organisation but you may not have permissions to add a project to
+the organisation. That's not a problem, you can configure it anyway and ask wg-operations to transfer it to
+the organisation afterwards (you have 100 free builds)
+
+## Rails specific setup
+
+*Setup Job:*
+
+    git checkout -- .ruby-version
+    export RAILS_ENV=test
+    export TZ=Europe/Zurich
+    bundle install --without production development --deployment --jobs 3 --retry 3
+    nvm use 10.10
```

Comment:
i would use a placeholder here, something like `<LATEST_LTS_VERSION_NUMBER>`

Reply:
Agree. I add also a link to the page where you can check it.